### ISSUE TITLE
Skype: Encrypted text (sometimes) not put into input field 

### ISSUE THREAD

**Issue by oversecio** [oversecio/oversec_crypto#17]
seems to happen only with very recent versions of Skype.

**Comment by oversecio** [comment]
Also seems to happen only after showing the encryption parameters activity. Subsequent encrypts are working fine.,

**Comment by oversecio** [comment]
correction, this is only happens with older versions (7.13.x),
so wontfix
